Source/Parallelization/GuardCellManager.cpp

```diff
@@ -145,7 +144,7 @@ guardCellManager::Init(
     ng_FieldSolverF = IntVect(AMREX_D_DECL(1,1,1));
 #endif
 
-    if (safe_guard_cells){
+    if (exchange_all_guard_cells){
```

Comment:
Could you explain why this had to be modified too?

Reply:
Thanks for the review!